pandas/core/arrays/timedeltas.py

```diff
@@ -620,10 +616,6 @@ def __floordiv__(self, other):
             if self.freq is not None:
                 # Note: freq gets division, not floor-division
                 freq = self.freq / other
-                if freq.nanos == 0 and self.freq.nanos != 0:
-                    # e.g. if self.freq is Nano(1) then dividing by 2
-                    #  rounds down to zero
-                    freq = None
```

Comment:
is reverting this necessary?  im surprised there isnt a test that hits it

Reply:
no idea. maybe added to stop failing tests with inferred freq?